repl/src/main/scala/ammonite/repl/interp/Interpreter.scala

```diff
@@ -33,10 +40,14 @@ class Interpreter(prompt0: Ref[String],
                   replArgs: Seq[Bind[_]])
   extends ImportHook.InterpreterInterface{ interp =>
 
-
   val hardcodedPredef =
     "import ammonite.repl.frontend.ReplBridge.repl.{pprintConfig, derefPPrint}"
 
+  // Should script as a whole be cached or not
+  val scriptCaching = storage match {
```

Comment:
This should not be necessary, as described in the comment below. `Storage.InMemory` should be able to cache things just fine, in memory.